File: VBdotNet2010/PDFFormFields/FormCombobox.vb
Public Class FormCombobox
    Public Property FieldIndex() As Integer
        Get
            Return Val(lblindex.Text)
        End Get

        Set(ByVal value As Integer)

            lblindex.Text = value.ToString()
        End Set

    End Property


    Public Property ComboBoxIndex() As Integer
        Get
            Return Val(txtcomboboxindex.Text)
        End Get

        Set(ByVal value As Integer)

            txtcomboboxindex.Text = value.ToString()
        End Set

    End Property

    Private Sub FormCombobox_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load

    End Sub

    Private Sub button1_Click(sender As System.Object, e As System.EventArgs) Handles button1.Click
        DialogResult = DialogResult.OK
        Close()
    End Sub
End Class

File: c#2010/PDFFormFields/FormCombobox.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace WindowsApplication1
{
    public partial class FormCombobox : Form
    {
        public FormCombobox()
        {
            InitializeComponent();
        }

        public int FieldIndex
        {
            set { lblindex.Text = value.ToString(); }
        }

        public int ComboBoxIndex
        {
            get { return Convert.ToInt32(txtcomboboxindex.Text); }
            set { txtcomboboxindex.Text = value.ToString(); }
        }

        private void button1_Click(object sender, EventArgs e)
        {
            this.DialogResult = DialogResult.OK;
            this.Close();
        }

        private void FormCombobox_Load(object sender, EventArgs e)
        {

        }
    }
}
